src/FhirLoader.Common/SourceFileHandler.cs

```diff
@@ -69,13 +70,58 @@ public IEnumerable<FhirFileHandler> LoadFromFilePath(string bundlePath, int bund
 
                 if (filePath.EndsWith(".json"))
                 {
-                    yield return new BundleFileHandler(safeFileStream, filePath, bundleSize, _logger);
+                    if (CheckResourceType(filePath)?.ToLower() == "bundle")
```

Comment:
Ideally, the CLI could load single resources or bundles. May be out of scope for this story, but that's the future goal. 😄